Example/Tests/UserDefaultsStorageTest.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  UserDefaultsStorageTest.swift
+//  Analytics
+//
+//  Copyright © 2016 Segment. All rights reserved.
+//
+
+import Quick
+import Nimble
+import Analytics
+
+class UserDefaultsStorageTest : QuickSpec {
+  override func spec() {
+    var storage : SEGUserDefaultsStorage!
+    beforeEach {
+//      let crypto = SEGAES256Crypto(password: "thetrees")
```

Comment:
It was actually kind of useful because it allowed changing the storage configuration and manually re-run the tests. Now we do have tests for multiple storage configuration, but ideally I'd love to have parametric test, where the same test suite is run but with different parameters. Not sure how to do them, which is the only reason I added a few sanity tests at the bottom (should work with namespace, should work with crypto, etc).